python/lsst/pipe/base/pipelineIR.py

```diff
@@ -651,6 +670,14 @@ def process_args(argument: str | dict) -> dict:
                     argument["include"] = [argument["include"]]
                 if "instrument" in argument and argument["instrument"] == "None":
                     argument["instrument"] = None
+                if "labeledSubsetModifyMode" in argument:
+                    match argument["labeledSubsetModifyMode"]:
+                        case "DROP":
+                            argument["labeledSubsetModifyMode"] = PipelineSubsetCtrl.DROP
+                        case "EDIT":
+                            argument["labeledSubsetModifyMode"] = PipelineSubsetCtrl.EDIT
+                        case unknown:
+                            raise ValueError(f"{unknown} is not a valid mode for labeledSubsetModifyMode")
```

Comment:
If you're in a unit-testing kind of mood, you could also add one to cover this raise? I won't object if you don't want to however.